darwin/future/tests/fixtures.py

```diff
@@ -0,0 +1,15 @@
+import os
+import shutil
+from pathlib import Path
+from typing import Generator
+
+import pytest
+
+
+@pytest.fixture
+def test_directory() -> Generator[Path, None, None]:
+    path = Path.home() / "darwin-tests"
```

Comment:
not sure why, but I couldn't get tests to create a hidden folder, ".darwin-tests" here, with permission denied exceptions. I think a mac thing. Non-hidden folders were fine.